### DISCUSSION TITLE
How to make items not able to be moved

### DISCUSSION THREAD

**Discussion by okay1204** [stefvanschie/IF#222] [Q&A] [answered]
I'm using a PatternPane to make a gray stained glass pane border, however, players can pick up these items and drag it into their inventory. How do I prevent this?

**Comment by okay1204** [comment, marked as answer]
Is `event.setCancelled(true)` correct?

**Reply by stefvanschie** [reply]
Yes, cancelling the click event is the correct way to prevent players from picking up items.
